packages/now-static-build/src/frameworks.ts

```diff
@@ -80,7 +80,7 @@ const frameworkList: Framework[] = [
   },
   {
     name: 'Docusaurus',
```

Comment:
```suggestion
    name: 'Docusaurus 2',
```

Lets make this consistent with framemworks.json